src/main/scala/com/softwaremill/realworld/utils/AbstractEndpoints.scala

```diff
@@ -0,0 +1,34 @@
+package com.softwaremill.realworld.utils
+
+import com.softwaremill.realworld.auth.{AuthService, UserSession}
+import com.softwaremill.realworld.db.{Db, DbConfig}
+import com.softwaremill.realworld.utils.*
+import io.getquill.SnakeCase
+import sttp.model.StatusCode
+import sttp.tapir.generic.auto.*
+import sttp.tapir.json.zio.jsonBody
+import sttp.tapir.server.ServerEndpoint.Full
+import sttp.tapir.ztapir.*
+import sttp.tapir.{EndpointInput, PublicEndpoint, Validator}
+import zio.json.{DeriveJsonDecoder, DeriveJsonEncoder}
+import zio.{Cause, Exit, ZIO, ZLayer}
+
+abstract class AbstractEndpoints(authService: AuthService):
+
+  val secureEndpoint: ZPartialServerEndpoint[Any, String, UserSession, Unit, ErrorInfo, Unit, Any] = endpoint.get
```

Comment:
does it have to be a `GET` endpoint?

Reply:
Right, I missed it because we have only get endpoints so far :)